roles/agnosticv/templates/governor.yaml.j2

```diff
@@ -68,84 +50,30 @@ spec:
 {% endfor %}
 
   subjectEventHandlers:
-    create:
-      roles:
-      - role: babylon_anarchy_governor
-    update:
+{% for __event in ('create', 'update', 'delete') %}
+    {{ __event }}:
       roles:
-      - role: babylon_anarchy_governor
-    delete:
-      roles:
-      - role: babylon_anarchy_governor
+{%   for __role in babylon_anarchy_roles %}
+      - role: {{ __role.name }}
+{%   endfor %}
+{% endfor %}
 
   actions:
-    # anarchy_action_name
-    provision:
-      roles:
-{% if merged_vars.__meta__.aws_sandboxed | default(false) %}
-      - role: babylon_aws_sandbox
-{% endif %}
-{% if merged_vars.__meta__.scheduler.enable | default(false) %}
-      - role: babylon_agnostics
-{% endif %}
-      - role: babylon_anarchy_governor
-      callbackHandlers:
-        # anarchy_action_name
-        # anarchy_action_callback_name
-        # anarchy_action_callback_data
-        started:
-          roles:
-{% if gpte_reporting %}
-          - role: babylon_gpte_reporting
-{% endif %}
-          - role: babylon_anarchy_governor
-        complete:
-          roles:
-{% if gpte_reporting %}
-          - role: babylon_gpte_reporting
-{% endif %}
-          - role: babylon_anarchy_governor
-    destroy:
-      roles:
-{% if merged_vars.__meta__.aws_sandboxed | default(false) %}
-      - role: babylon_aws_sandbox
-{% endif %}
-{% if merged_vars.__meta__.scheduler.enable | default(false) %}
-      - role: babylon_agnostics
-{% endif %}
-      - role: babylon_anarchy_governor
-      callbackHandlers:
-        started:
-          roles:
-          - role: babylon_anarchy_governor
-        complete:
-          roles:
-{% if merged_vars.__meta__.aws_sandboxed | default(false) %}
-          - role: babylon_aws_sandbox
-{% endif %}
-{% if merged_vars.__meta__.scheduler.enable | default(false) %}
-          - role: babylon_agnostics
-{% endif %}
-          - role: babylon_anarchy_governor
-    # Check provision status to detect failures
-    check:
-      roles:
-      - role: babylon_anarchy_governor
-{% for __action in ('start', 'stop', 'status') %}
+{% for __action in ('provision', 'destroy', 'start', 'stop', 'status', 'check') %}
     {{ __action }}:
       roles:
-{%   if merged_vars.__meta__.aws_sandboxed | default(false) %}
-      - role: babylon_aws_sandbox
-{%   endif %}
-{% if merged_vars.__meta__.scheduler.enable | default(false) %}
-      - role: babylon_agnostics
-{% endif %}
+{%   for __role in babylon_anarchy_roles %}
+      - role: {{ __role.name }}
+{%   endfor %}
       - role: babylon_anarchy_governor
```

Comment:
why is this line needed ?
it's already included with babylon_anarchy_roles, isn't it ?

Reply:
Ah, good catch!